core/frontend-stubs-testing/src/main/java/org/eclipse/sw360/antenna/frontend/testing/AbstractAntennaFrontendTest.java

```diff
@@ -66,8 +66,7 @@ protected void setRunExecutionTest(boolean runExecutionTest) {
                 {(Supplier<AbstractTestProjectWithExpectations>) MinimalTestProject::new, "minimal configuration"},
                 {(Supplier<AbstractTestProjectWithExpectations>) BasicTestProject::new, "basic configuration"},
                 {(Supplier<AbstractTestProjectWithExpectations>) ExampleTestProject::new, "example configuration"},
-                {(Supplier<AbstractTestProjectWithExpectations>) MavenTestProject::new, "maven configuration"},
-                {(Supplier<AbstractTestProjectWithExpectations>) P2TestProject::new, "p2 configuration"}
```

Comment:
If we keep the p2 example tests, we should also keep the Test classes for it.

Reply:
Done.